samples/client/petstore/dart2/petstore_client_lib/lib/api/user_api.dart

```diff
@@ -424,7 +424,7 @@ class UserApi {
     // At the time of writing this, `dart:convert` will throw an "Unexpected end of input"
     // FormatException when trying to decode an empty string.
     if (response.body != null && response.statusCode != HttpStatus.noContent) {
-      return apiClient.deserialize(_decodeBodyBytes(response), 'String') as String;
+      return String.fromJson(json.decode(response.body));
```

Comment:
There are a couple of these

Reply:
yea I know, no change here, it didnt compile, it doesnt compile now, just in a different way

Reply:
ok, i found `{{#isPrimitiveType}}`

Reply:
`isPrimitiveType` doesnt work yet...  I'll look into it tomorrow

Reply:
`returnTypeIsPrimitive`